AUTHORS

```diff
@@ -433,6 +434,7 @@ Cristian Staretu <cristian.staretu@gmail.com>
 cristiano balducci <cristiano.balducci@gmail.com>
 Cristina Yenyxe Gonzalez Garcia <cristina.yenyxe@gmail.com>
 Cruceru Calin-Cristian <crucerucalincristian@gmail.com>
+cui fliter <imcusg@gmail.com>
 CUI Wei <ghostplant@qq.com>
 cuishuang <imcusg@gmail.com>
```

Comment:
:see_no_evil:

Reply:
Good eye

Reply:
oh! nice one. usually I'm spotting these. Did actually check git log for one of the other ones to see if we had a "correct" name 😂

Reply:
Nope, just noticed it in the diff while reviewing in the web UI -- I imagine there could be others, but I was not thorough :sweat_smile: